pkg/util/csi/volume_snapshot.go

```diff
@@ -47,59 +48,89 @@ const (
 )
 
 // WaitVolumeSnapshotReady waits a VS to become ready to use until the timeout reaches
-func WaitVolumeSnapshotReady(ctx context.Context, snapshotClient snapshotter.SnapshotV1Interface,
-	volumeSnapshot string, volumeSnapshotNS string, timeout time.Duration, log logrus.FieldLogger) (*snapshotv1api.VolumeSnapshot, error) {
+func WaitVolumeSnapshotReady(
+	ctx context.Context,
+	snapshotClient snapshotter.SnapshotV1Interface,
+	volumeSnapshot string,
+	volumeSnapshotNS string,
+	timeout time.Duration,
+	log logrus.FieldLogger,
+) (*snapshotv1api.VolumeSnapshot, error) {
 	var updated *snapshotv1api.VolumeSnapshot
 	errMessage := sets.NewString()
 
-	err := wait.PollUntilContextTimeout(ctx, waitInternal, timeout, true, func(ctx context.Context) (bool, error) {
-		tmpVS, err := snapshotClient.VolumeSnapshots(volumeSnapshotNS).Get(ctx, volumeSnapshot, metav1.GetOptions{})
-		if err != nil {
-			return false, errors.Wrapf(err, fmt.Sprintf("error to get volumesnapshot %s/%s", volumeSnapshotNS, volumeSnapshot))
-		}
+	err := wait.PollUntilContextTimeout(
+		ctx,
+		waitInternal,
+		timeout,
+		true,
+		func(ctx context.Context) (bool, error) {
+			tmpVS, err := snapshotClient.VolumeSnapshots(volumeSnapshotNS).Get(
+				ctx, volumeSnapshot, metav1.GetOptions{})
+			if err != nil {
+				return false, errors.Wrapf(
+					err,
+					fmt.Sprintf("error to get VolumeSnapshot %s/%s",
+						volumeSnapshotNS, volumeSnapshot),
+				)
+			}
 
-		if tmpVS.Status == nil {
-			return false, nil
-		}
+			if tmpVS.Status == nil {
+				return false, nil
+			}
 
```

Comment:
Not sure whether it has an indent problem?

Reply:
No. This is caused by how the function `wait.PollUntilContextTimeout` is called.

Before modification, the function's parameters are placed into one line.
`err := wait.PollUntilContextTimeout(ctx, waitInternal, timeout, true, func(ctx context.Context) (bool, error) {`

After modification, the functions' parameters are split into different lines.
``` go
	err := wait.PollUntilContextTimeout(
		ctx,
		waitInternal,
		timeout,
		true,
		func(ctx context.Context) (bool, error) {
			tmpVS, err := snapshotClient.VolumeSnapshots(volumeSnapshotNS).Get(
				ctx, volumeSnapshot, metav1.GetOptions{})
```

That caused the intent change.